models/classes/menu/SectionVisibilityFilter.php

```diff
@@ -23,16 +23,21 @@
 namespace oat\tao\model\menu;
 
 use LogicException;
+use oat\generis\model\GenerisRdf;
 use oat\oatbox\service\ConfigurableService;
 use oat\tao\model\featureFlag\FeatureFlagChecker;
 use oat\tao\model\featureFlag\FeatureFlagCheckerInterface;
+use oat\tao\model\user\implementation\UserSettingsService;
+use oat\tao\model\user\UserSettingsInterface;
 
 class SectionVisibilityFilter extends ConfigurableService implements SectionVisibilityFilterInterface
 {
     public const SERVICE_ID = 'tao/SectionVisibilityFilter';
     public const OPTION_FEATURE_FLAG_SECTIONS = 'featureFlagSections';
     public const OPTION_FEATURE_FLAG_SECTIONS_TO_HIDE = 'featureFlagSectionsToHide';
 
+    //TODO:: Add sections hidden for simple mode
+    public const SIMPLE_MODE_HIDDEN_SECTIONS = [];
```

Comment:
We must separate constants from other sections by black line.
```suggestion
    public const SIMPLE_MODE_HIDDEN_SECTIONS = [];
    
```